pkg/scheduler/core/generic_scheduler.go

```diff
@@ -1208,6 +1213,7 @@ func NewGenericScheduler(
 	alwaysCheckAllPredicates bool,
 	disablePreemption bool,
 	percentageOfNodesToScore int32,
+	enableNonPreempting bool,
```

Comment:
We should probably create a struct for all of these configuration related parameters and pass that struct as an argument to `NewGenericScheduler`, but not in this PR. We can do that in a follow-up PR.

Reply:
+1